### ISSUE TITLE
about the feature extraction

### ISSUE THREAD

**Issue by jackyyigao** [Xiaobin-Rong/deepvqe#8]
Hi,  Xiaobin, the FE block in you implementation seems a kind of normalization to the complex spectra, could you help introduce the paper to describle it? Seems it's not the _power law compressed complex spectra_ as mentioned in the deepVQE paper?  thanks for any info.

**Comment by Xiaobin-Rong** [comment]
Hi, I think the codes below actually play the role of power law compressing.
```
x_mag = torch.sqrt(x[...,[0]]**2 + x[...,[1]]**2 + 1e-12)
x_c = torch.div(x, x_mag.pow(1-self.c) + 1e-12)
```
Actually, the codes are equal to such a version, which may be more intuitional.
```
x_mag = torch.sqrt(x[...,[0]]**2 + x[...,[1]]**2 + 1e-12)
x_c_mag = x_mag.pow(self.c)
x_c_real = x_c_mag * x[...,[0]] / (x_mag + 1e-12)
x_c_imag = x_c_mag * x[...,[1]] / (x_mag + 1e-12)
x_c = torch.cat([x_c_real, x_c_imag], dim=-1)
```
I hope I have clarified your confuse, If not, feel free to ask me.

**Comment by jackyyigao** [comment]
thank you very much for your so quick response! BTW, your latest work on GTCRN seems very promising for real-time implementation for commercial products. hope could will have change to communicate with you later.

**Comment by Xiaobin-Rong** [comment]
Thank you for your appreciation, and I'm glad to keep in touch with you
